src/test_l1_dsHdmiIn.c

```diff
@@ -535,87 +542,85 @@ void test_l1_dsHdmiIn_negative_dsHdmiInSelectPort(void) {
     // Step 2: Initialize the HDMI input sub-system using dsHdmiInInit()
     UT_ASSERT_EQUAL_FATAL(dsHdmiInInit(), dsERR_NONE);
 
-    // Step 3: Get the number of input ports from the profile file
     numInputPorts = UT_KVP_PROFILE_GET_UINT8("dsHdmiIn/HdmiInputPort/numberOfPorts");
     for (uint8_t i = dsHDMI_IN_PORT_0; i < numInputPorts; i++)
     {
-	  // Step 4: Call dsHdmiInSelectPort() with an invalid value
+	  // Step 3: Call dsHdmiInSelectPort() with an invalid value
```

Comment:
for negative cases, we dont require loops, 

    // Step 3: Call dsHdmiInSelectPort() with an invalid value
    UT_ASSERT_EQUAL(dsHdmiInSelectPort(dsHDMI_IN_PORT_MAX, false, dsVideoPlane_PRIMARY, false), dsERR_INVALID_PARAM);

    // Step 4: Call dsHdmiInSelectPort() with an invalid plane value
    UT_ASSERT_EQUAL(dsHdmiInSelectPort(dsHDMI_IN_PORT_0, false, dsVideoPlane_MAX, false), dsERR_INVALID_PARAM);

keep the original code